Additional troubleshooting for Sentry ASP .NET (#4407)
getsentry/sentry-docs: src/platforms/dotnet/guides/aspnet/troubleshooting.mdx

ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12
```

> ### SentrSdk.Init with error

Reviewer: ```suggestion
### SentrSdk.Init With Error
```

> In case you receive the following error during Sentry initialization:

Reviewer: ```suggestion
If you get the following error during Sentry initialization, you may have a misconfiguration on your Web.config:
```

`System.IO.FileNotFoundException: Could not load file or assembly System.Net.Http, Version=4.0.0.0`

> You may have a misconfiguration on your Web.config. You may fix it with one of the three solutions below:

Reviewer: ```suggestion
You can fix it with one of the three solutions below:
```

* Find the `assemblyIdentity` for `System.Net.Http` and remove the `bindingRedirect`.
```xml
<dependentAssembly>
  <assemblyIdentity name="System.Net.Http" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
</dependentAssembly>
```

> * Find the `assemblyIdentity` for `System.Net.Http` and lower the newVersion to `4.0.0.0`

Reviewer: ```suggestion
* Find the `assemblyIdentity` for `System.Net.Http` and lower the newVersion to `4.0.0.0`.
```

```xml
<dependentAssembly>
  <assemblyIdentity name="System.Net.Http" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
  <bindingRedirect oldVersion="0.0.0.0-4.2.0.0" newVersion="4.0.0.0" />
</dependentAssembly>
```

* Install the latest package from `System.Net.Http` in NuGet into your project.

> More information about the problem can be found here: https://github.com/dotnet/runtime/issues/24382

Reviewer: ```suggestion
You can find more information about the problem [in this GitHub issue] (https://github.com/dotnet/runtime/issues/24382).
```

ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12
```

### SentrSdk.Init With Error

If you get the following error during Sentry initialization, you may have a misconfiguration on your Web.config:

`System.IO.FileNotFoundException: Could not load file or assembly System.Net.Http, Version=4.0.0.0`

You can fix it with one of the three solutions below:

* Find the `assemblyIdentity` for `System.Net.Http` and remove the `bindingRedirect`.
```xml
<dependentAssembly>
  <assemblyIdentity name="System.Net.Http" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
</dependentAssembly>
```

* Find the `assemblyIdentity` for `System.Net.Http` and lower the newVersion to `4.0.0.0`.
```xml
<dependentAssembly>
  <assemblyIdentity name="System.Net.Http" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
  <bindingRedirect oldVersion="0.0.0.0-4.2.0.0" newVersion="4.0.0.0" />
</dependentAssembly>
```

* Install the latest package from `System.Net.Http` in NuGet into your project.

> You can find more information about the problem [in this GitHub issue] (https://github.com/dotnet/runtime/issues/24382).

Reviewer 2: ```suggestion
You can find more information about the problem [in this GitHub issue](https://github.com/dotnet/runtime/issues/24382).
```
